lib/error-handler.js

```diff
@@ -4,9 +4,8 @@ const Integrations = require('@sentry/integrations')
 
 Sentry.init({
   dsn: process.env.SENTRY_DSN,
-  release: process.env.HEROKU_SLUG_COMMIT
-    ? process.env.HEROKU_SLUG_COMMIT
-    : undefined,
+  environment: process.env.NODE_ENV,
+  release: process.env.HEROKU_SLUG_COMMIT,
```

Comment:
I tweaked this because I believe `null` results in the same behaviour from Sentry.